lib/libesp32_lvgl/lv_binding_berry/src/embedded/lvgl_ctypes.py

```diff
@@ -35,6 +35,7 @@
 lv_dir = ct.u8
 lv_coord_t = ct.i32
 lv_value_precise = ct.i32
+lv_style_prop_t = ct.u8
```

Comment:
This is not actually useful

Reply:
Removed it, you are right, it is not of any use.